sqlmesh/core/model/seed.py

```diff
@@ -39,7 +39,14 @@ def _bool_validator(cls, v: t.Any) -> t.Optional[bool]:
     def _str_validator(cls, v: t.Any) -> t.Optional[str]:
         if v is None or not isinstance(v, exp.Expression):
             return v
-        return v.this
+
+        v = v.this
+        if v == "\\t":
```

Comment:
Do we need to take into account other escaped characters like `\r`, `\n`, etc?

Reply:
Yeah will amend by doing a quick sqlglot refactor.

Reply:
Addressed via https://github.com/tobymao/sqlglot/commit/89a8984b8db3817d934b4395e190f3848b1ee77a and https://github.com/TobikoData/sqlmesh/pull/2690/commits/f6429761f8ed66f3855bf6030e5edba26a2c23c0.